wadas/domain/operation_mode.py

```diff
@@ -173,16 +193,17 @@ def send_notification(self, detection_event: DetectionEvent, message):
         """Method to send notification(s) trough Notifier class (and subclasses)"""
         Notifier.send_notifications(detection_event, message)
 
-    def actuate(self, camera_id):
+    def actuate(self, detection_event: DetectionEvent):
         """Method to trigger actuators associated to the camera, when enabled"""
         cur_camera = None
         for cur_camera in cameras:
-            if cur_camera.id == camera_id:
+            if cur_camera.id == detection_event.camera_id:
                 break
         if cur_camera:
             for actuator in cur_camera.actuators:
                 if actuator.enabled:
-                    actuator.actuate()
+                    actuation_event = ActuationEvent(actuator.id, get_timestamp(), detection_event)
+                    actuator.actuate(actuation_event)
```

Comment:
Move code in L203-206 here, and after that you can break.
This way you can remove L198 and L202.